tests/containers/docker_image.pm

```diff
@@ -37,13 +38,16 @@ sub run {
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
 
     for my $iname (@{$image_names}) {
+        record_info 'image name', '$iname';
         test_container_image(image => $iname, runtime => $runtime);
         build_container_image(image => $iname, runtime => $runtime);
```

Comment:
Not part of this PR, but we should not forget that building **SLE** ***containers***  is supported only on **SLE** ***host*** atm. IMHO this should be another test case/module. WDYT?

Reply:
my understanding was that all this cases should be tackled by the same module. If you mean to break it down a bit i am with you. But this a discussion with all the team in which direction will go. For sure some things should become more clear, as i feel that we overload the functions to handle all the cases.

Reply:
Yeah, at some point it seemed like the way to go, and we should talk about this someday again as our strategy is still evolving. Nevertheless, I would say it is not so blurry as it was, so it would be nice if we can find some time for a virtual coffee meeting, so we can discuss this briefly.